README.md

```diff
@@ -1,8 +1,7 @@
-# RaspberryCast 3.0
+# RaspberryCast 3.0 [![Build Status](https://travis-ci.org/mheine/RaspberryCast.svg?branch=travis)](https://travis-ci.org/mheine/RaspberryCast)
```

Comment:
By hasard, won't it show the build status of your fork instead of the main project? Or is it related to your account connected to this repo?